tests/vm_test.go

```diff
@@ -436,69 +441,68 @@ var _ = Describe("VirtualMachine", func() {
 				var vmi *v1.VirtualMachineInstance
 				var err error
-				By("getting an VMI")
-				newVMI := newVirtualMachine(false)
+				By("getting an VM")
+				newVM := newVirtualMachine(false)
 
 				By("Invoking virtctl start")
-				virtctl := tests.NewRepeatableVirtctlCommand(vm.COMMAND_START, "--namespace", newVMI.Namespace, newVMI.Name)
+				virtctl := tests.NewRepeatableVirtctlCommand(vm.COMMAND_START, "--namespace", newVM.Namespace, newVM.Name)
 
 				err = virtctl()
-				Expect(err).ToNot(HaveOccurred())
+				Expect(err).ToNot(HaveOccurred(), "Should execute virtctl correctly")
 
-				By("Getting the status of the VMI")
+				By("Getting the status of the VM")
 				Eventually(func() bool {
-					newVMI, err = virtClient.VirtualMachine(newVMI.Namespace).Get(newVMI.Name, &v12.GetOptions{})
-					Expect(err).ToNot(HaveOccurred())
-					return newVMI.Status.Ready
-				}, 360*time.Second, 1*time.Second).Should(BeTrue())
+					newVM, err = virtClient.VirtualMachine(newVM.Namespace).Get(newVM.Name, &v12.GetOptions{})
+					Expect(err).ToNot(HaveOccurred(), "Should retrieve new VM")
+					return newVM.Status.Ready
+				}, 120*time.Second, 5*time.Second).Should(BeTrue(), "Retrieved VM should be ready")
 
 				By("Getting the running VirtualMachineInstance")
 				Eventually(func() bool {
-					vmi, err = virtClient.VirtualMachineInstance(newVMI.Namespace).Get(newVMI.Name, &v12.GetOptions{})
-					Expect(err).ToNot(HaveOccurred())
+					vmi, err = virtClient.VirtualMachineInstance(newVM.Namespace).Get(newVM.Name, &v12.GetOptions{})
+					Expect(err).ToNot(HaveOccurred(), "Should retrieve VMI")
 					return vmi.Status.Phase == v1.Running
-				}, 240*time.Second, 1*time.Second).Should(BeTrue())
+				}, 30*time.Second, 5*time.Second).Should(BeTrue(), "Retrieved VMI should be running")
 
 				By("Ensuring a second invocation should fail")
 				err = virtctl()
-				Expect(err).To(HaveOccurred())
+				Expect(err).To(HaveOccurred(), "Calling virtctl second time should fail")
 			})
 
 			It("should stop a VirtualMachineInstance once", func() {
 				var err error
-				By("getting an VMI")
-				newVMI := newVirtualMachine(true)
+				By("getting an VM")
+				newVM := newVirtualMachine(true)
 
 				By("Invoking virtctl stop")
-				virtctl := tests.NewRepeatableVirtctlCommand(vm.COMMAND_STOP, "--namespace", newVMI.Namespace, newVMI.Name)
+				virtctl := tests.NewRepeatableVirtctlCommand(vm.COMMAND_STOP, "--namespace", newVM.Namespace, newVM.Name)
 
-				By("Ensuring VMI is running")
-				Eventually(func() bool {
-					newVMI, err = virtClient.VirtualMachine(newVMI.Namespace).Get(newVMI.Name, &v12.GetOptions{})
-					Expect(err).ToNot(HaveOccurred())
-					return newVMI.Status.Ready
-				}, 360*time.Second, 1*time.Second).Should(BeTrue())
+				By("Ensuring VM is running")
+				Eventually(func() error {
+					_, err = virtClient.VirtualMachine(newVM.Namespace).Get(newVM.Name, &v12.GetOptions{})
+					return err
```

Comment:
This test is no longer accurate with these changes.

checking for a GET error does not occur is not the same as ensuring newVMI.Status.Ready == true.  This is just checking that the VM exists, not that it's running. The previous check was right